### ISSUE TITLE
Tests failed for GAE

### ISSUE THREAD

**Issue by Zevran** [bugsnag/bugsnag-go#19]
Hello,

I am using your package for google app engine and since I update google cloud sdk I got this error :

```
goapp test
# github.com/bugsnag/bugsnag-go
./appengine.go:44: config.Logger.Prefix undefined (type interface { Printf(string, ...interface {}) } has no field or method Prefix)
./appengine.go:44: config.Logger.Flags undefined (type interface { Printf(string, ...interface {}) } has no field or method Flags)
```

But when I do the same tests with standard go binary I got no errors :

```
2015/05/13 22:12:57 notifying bugsnag: hello world
2015/05/13 22:12:57 notifying bugsnag: eggs
2015/05/13 22:12:57 notifying bugsnag: ham
configuration_test.go:99: hello bugsnag
2015-05-13 20:12:57 WARNING test configuration_test.go:69 hello bugsnag
PASS
ok  	github.com/bugsnag/bugsnag-go	0.076s
```

Can someone help me ?

Thanks

**Comment by Zevran** [comment]
I just changed the line 44 by :
```
config.Logger = log.New(appengineWriter{c}, log.Prefix(), log.Flags())
```
Build pass but test stops on bugsnag.Notify :
```
goapp test
2015/05/13 22:49:50 appengine: not running under devappserver2; using some default configuration
2015/05/13 22:49:50 bugsnag.Notify: No appengine context given
```

**Comment by ConradIrwin** [comment]
Thanks for bringing this up, @Zevran. It should be fixed by `v1.0.4` which I've just pushed to github.

Some of the changes made in `v1.0.3` were not tested on app-engine.
